src/Form/listenForFormSubmission.js

```diff
@@ -1,40 +1,25 @@
 const {forms} = require('../scanForInputs')
 const isApp = require('../autofill-utils')
 
-const submitHandler = () => {
-    if (!forms.size) return
-
-    const filledForm = [...forms.values()].find(form => form.hasValues())
-    filledForm?.submitHandler()
-}
-
-let listening = false
-
 const listenForGlobalFormSubmission = () => {
-    if (listening || !isApp) return
-
-    window.addEventListener('submit', submitHandler, true)
-
-    // This is rather heavy-handed and likely to capture false-positives
-    // TODO: use a better solution for capturing events rather than this unload event
-    window.addEventListener('unload', submitHandler)
+    if (!isApp) return
 
     try {
         const observer = new PerformanceObserver((list) => {
             const entries = list.getEntries().filter((entry) =>
-                entry.initiatorType === 'xmlhttprequest' &&
-                entry.name.split('?')[0].match(/login|sign-in|signin|session/)
+                ['fetch', 'xmlhttprequest'].includes(entry.initiatorType) &&
+                entry.name.match(/login|sign-in|signin|session/)
```

Comment:
I am now checking both `fetch` and `xmlhttprequest` to catch Quora's logins.

I have also removed that `split('?')[0]` to scan the whole input. @brindy Do you know why it was added? Was it overtriggering otherwise?